### ISSUE TITLE
Wide characters are not properly supported

### ISSUE THREAD

**Issue by khs26** [zellij-org/zellij#429]
## Issue
When reading from a binary file (using less) some characters seem to overflow from the right edge of the screen to the left (appearing in another pane if using a vertical split).

## Repro steps
From a new zellij session:
1) Open a new pane (split right)
2) In the new pane, `less -f /dev/urandom`
3) Scroll until you see characters appearing on the left of the screen (in the other pane)

## Debug logs and info
```
kyle@ubuntu:/tmp/zellij$ tput cols
87
kyle@ubuntu:/tmp/zellij$ tput lines
43
kyle@ubuntu:/tmp/zellij$ uname -a                                                                                                                                               
Linux ubuntu 5.8.0-43-generic #49-Ubuntu SMP Fri Feb 5 03:01:28 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux 
```

Working off main 099861ff5b0f83773ca0af4c70e6e39be3b0336c

![pane-overflow](https://user-images.githubusercontent.com/3355128/116807018-a8bb1e00-aae5-11eb-8cfc-5ac74c7e4ce0.png)
[log.txt](https://github.com/zellij-org/zellij/files/6410986/log.txt)
[zellij-3.log](https://github.com/zellij-org/zellij/files/6410987/zellij-3.log)
[zellij-4.log](https://github.com/zellij-org/zellij/files/6410988/zellij-4.log)
[zellij-7.log](https://github.com/zellij-org/zellij/files/6410989/zellij-7.log)
[zellij-13.log](https://github.com/zellij-org/zellij/files/6410990/zellij-13.log)

**Comment by horasal** [comment]
This is a widechar related issue. Now line width is not correctly counted if it contains widechar(emoji, cjk characters, etc.).

**Comment by DumboJet** [comment]
A bit irrelevant, but:
Anyone knows why the "arrow" characters on the top and bottom menus of Zellij appear broken and shown as squares, as shown in the above screenshot?
I guess it is a font issue, but I have no idea how to fix it.
Shouldn't this work out-of-the-box?

Really nice app by the way!
I hate having to remember shortcuts (or commands for that matter...) by-heart and Zellij is perfect for avoiding this.
Although, I am still trying to figure out how to escape from entering the pane full-screen mode...
But it is the 1st day trying Zellij, so... I am sure I will find out some time...

**Comment by a-kenji** [comment]
@DumboJet 

Hey! No problem. It most likely is a missing icon of your font. 
I hope this helps:
https://zellij.dev/documentation/compatibility.html

**Comment by imsnif** [comment]
Fixed in `0.12.0`
